### DISCUSSION TITLE
Add Horizontal Line in Settings>Data Models

### DISCUSSION THREAD

**Discussion by Dominic-Preap** [directus/directus#19951] [Feature Requests]
### Summary

I would be nice to have Horizontal Line in Data Models for separating the menu along side with folder

![image](https://github.com/directus/directus/assets/14802170/28d8bafe-abd7-46fe-9774-62253683d992)

### Basic Example

The above image

### Motivation

It would be more clean of for slidebar UI

### Detailed Design

We can using **Create Folder** but have option to choose Horizonal line or Folder.

![image](https://github.com/directus/directus/assets/14802170/768c08cb-1330-499e-b900-4db5905c88df)

### Requirements List

None

### Drawbacks

None

### Alternatives

None

### Adoption Strategy

HR is already existing, we can leverage this component

![image](https://github.com/directus/directus/assets/14802170/d44b1ae7-6159-4852-b841-900b0e35d350)

**Comment by Seinnd** [comment]
I may be wrong, but I think the current structure does not provide (presumably for security reasons) for empty folders to be seen by non-admin users.

So for the time being you can achieve this with a single line of CSS at your 'project settings', just by assign all parent groups an appropriate pseudo-element for example:

`#navigation .content-navigation > .v-list-group::before { content:""; display:inline-block; width:100%; height:0; margin:1em 0; outline:1px solid currentColor; opacity:0.1; }`

or other creative ways like the DOM position of the element or data attributes like the assigned icon (and so on) to assign this approach to a single element either.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
